google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/wrappers/Row.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.wrappers;
+
+import com.google.api.core.BetaApi;
+import com.google.api.core.InternalApi;
+import com.google.auto.value.AutoValue;
+import com.google.protobuf.ByteString;
+import java.util.List;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/** Default representation of a logical row. */
+@BetaApi
+@AutoValue
+public abstract class Row implements Comparable<Row> {
+  /** Creates a new instance of the {@link Row}. */
+  @InternalApi
+  public static Row create(ByteString key, List<Cell> cells) {
+    return new AutoValue_Row(key, cells);
+  }
+
+  /** Returns the row key */
+  @Nonnull
+  public abstract ByteString key();
```

Comment:
The general convention in google-cloud-java is to use getX()/setX()

Reply:
For immutable classes, I think that the get prefix creates unnecessary noise. But I don't have strong feelings on it, so I changed it